ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -0,0 +1,36 @@
+# global
+import ivy
+
+
+def _compute_allclose_with_tol(input, other, rtol, atol):
+    return ivy.all(
+        ivy.less_equal(
+            ivy.abs(ivy.subtract(input, other)),
+            ivy.add(atol, ivy.multiply(rtol, ivy.abs(other))),
+        )
+    )
+
+
+def allclose(input, other, rtol=1e-05, atol=1e-08, equal_nan=False):
+    finite_input = ivy.isfinite(input)
+    finite_other = ivy.isfinite(other)
+    if ivy.all(finite_input) and ivy.all(finite_other):
+        return _compute_allclose_with_tol(input, other, rtol, atol)
+    else:
```

Comment:
It might not be necessary to handle the case with infinites, does np.all_close() handle inf and -inf?

Reply:
I think if we have inf or -inf values the comparison will depend on equal_nan mainly that's why if we have some inf values in our array we will either test them with `equal_nan=True` or `equal_nan=False`  and  the `np.allclose()` handles inf values

Reply:
Ok, if np handles it then we should indeed manage inf and -inf :)